thundra/thundra_agent.py

```diff
@@ -46,6 +47,15 @@ def __init__(self, api_key=None, disable_trace=False, disable_metric=False, disa
         audit_response_skip_by_env = utils.get_environment_variable(constants.THUNDRA_LAMBDA_TRACE_RESPONSE_SKIP)
         self.response_skipped = utils.should_disable(audit_response_skip_by_env, response_skip)
 
+        is_warmup_aware_by_env = utils.get_environment_variable(constants.THUNDRA_LAMBDA_WARMUP_WARMUPAWARE)
+        self.warmup_aware = utils.should_disable(is_warmup_aware_by_env)
+
+        timeout_margin = utils.get_environment_variable(constants.THUNDRA_LAMBDA_TIMEOUT_MARGIN)
+        self.timeout_margin = int(timeout_margin) if timeout_margin is not None else 0
+        if self.timeout_margin is None or self.timeout_margin <= 0:
+            logger.warning('Timeout margin is set to default value(180) since nonpositive value cannot be given')
```

Comment:
I think you should get the '180' from the constants and append it to this logger.
`constants.DEFAULT_LAMBDA_TIMEOUT_MARGIN`
When we developer changes the value from the constants you will be giving wrong information from the logs. 


Also logging it after the assignment will be appropriate, what do you think?